src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -40,19 +41,21 @@
     private final CaseUpdateClient caseUpdateClient;
     private final ServiceResponseParser serviceResponseParser;
     private final ExceptionRecordFinalizer exceptionRecordFinalizer;
+    private final PaymentsProcessor paymentsProcessor;
```

Comment:
We already update payments for `SUPPLEMENTARY_EVIDENCE` type, using payment processor which is injected into `AttachCaseCallbackService` (which is the class calling this class).

Perhaps it could be done in a similar way here?
Ie, call the payment processor from `AttachCaseCallbackService` instead of adding this new dependency and code here. After all, this class is responsible for updating CCD.